### PR TITLE
Using go module instead of dep

### PR DESCRIPTION
**Pull request by koushki**
Related to https://github.com/Microsoft/ethr/issues/78 discussion.

### CODE REVIEW

**Comment by pgmsft** [comment]
@radu-matei can you review please?

**Comment by l0n3c0d3r** [comment]
@radu-matei  Would you be able to review this?

File: README.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ https://github.com/Microsoft/ethr/releases/latest
...
-Note: go version 1.10 or higher is required building it from the source.
+Note: go version 1.11 or higher is required building it from the source.
 
 ## Building from Source
 
 ```
 git clone https://github.com/Microsoft/ethr.git
 cd ethr
-dep ensure -v
+go get ./...
```
Comment (radu-matei) [inline README.md]: As this is a rather new feature in Go, I'd [point to the doc on how to use modules](https://github.com/golang/go/wiki/Modules#how-to-use-modules) and specify to set the `GO111MODULE=on` environment variable.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module github.com/Microsoft/ethr
+
+require (
+	github.com/mattn/go-runewidth v0.0.3
+	github.com/nsf/termbox-go v0.0.0-20181027232701-60ab7e3d12ed
+	golang.org/x/sys v0.0.0-20181206074257-70b957f3b65e
```
Comment (radu-matei) [inline go.mod]: Any particular reason why this is added here?

Comment (koushki) [reply go.mod]: we use this package somewhere in plt_darwin.go file, I made sure we have exact same revision as Gopkg.lock

File: README.md (hunk 1 above)
Comment (koushki) [reply README.md]: done!

File: go.mod (hunk 2 above)
Comment (radu-matei) [reply go.mod]: OK, thanks for the clarification!

**Pull request merged**